src/ext/java/org/opentripplanner/ext/readiness_endpoint/ActuatorAPI.java

```diff
@@ -18,34 +39,92 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ActuatorAPI.class);
 
+    public static final PrometheusMeterRegistry prometheusRegistry = new PrometheusMeterRegistry(
```

Comment:
Forgive my ignorance but what does it mean to have Prometheus registry in Java code? I thought that Prometheus is a metrics database. 

Is there a generic registry that works with other database, for example InfluxDB.

Or is this only about the serialization format being returned in the HTTP endpoint?

Reply:
This is about the serialization format of the responses